### PR TITLE
Initial Draft: Namecheap / Domain

### PR DESCRIPTION
**Pull request by alxmrs**
- Wrote first published draft of article "Getting Back my Domain"
- Adding "Drafts" section for articles in progress.

### CODE REVIEW

File: blog/2019-07-12-getting-back-my-domain.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+"How embarrassing. A software engineer who doesn't know how to set up HTTPS for his own personal site."
+
+When I started this site in 2017, that was the case. At least, I was focused on getting something up and running quick, 
+and snoozed on adding that feature. Like most of software engineering, building things outside of your current ability 
+is just a matter of Googling it and following directions. 
+
+As time has gone on, I've kept postponing this feature, despite 
+[learning how to implement it](https://github.blog/2018-05-01-github-pages-custom-domains-https/).
+I've avoided addressing anything to do with my domain due to an completely-avoidable predicament that I got
```
Comment (camerose) [inline blog/2019-07-12-getting-back-my-domain.md]: change completely-avoidable to completely avoidable

File: blog/2019-07-12-getting-back-my-domain.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+personal information. So I replaced a lot of information on various accounts with generated emails, cellphone numbers, 
+and credit cards. 
+
+Sometime around the end of high school or early college, after adopting MaskMe, I thought to myself,
+ "I should buy my personal domain before [another Alex Rosengarten](https://www.facebook.com/public/Alex-Rosengarten) 
+beats me to it."
+
+So I did. But I had the *brilliant* idea to make the account holding my domains as anonymous as possible. 
+You know, just in case I wanted to buy *other* domains where I would *need* anonymity (spoiler: this never happened). 
+I filled out my account information without my actual email, name, or phone number, but at least a real credit card.
```
Comment (camerose) [inline blog/2019-07-12-getting-back-my-domain.md]: change "at least" to "with"

File: blog/2019-07-12-getting-back-my-domain.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+
+Fast forward several years. 
+
+I somehow get into my domain name service (Namecheap) and wire up my domain to point to Github pages. I set up a quick, 
+custom jekyll blog, autopay for my domain every year, and never have to log into the domain name service for any reason. 
+
+Fast forward, later still.
+
+I've gaind maturity as a developer and know my way around cloud technologies (like GCP and AWS). I'm motivated to
```
Comment (camerose) [inline blog/2019-07-12-getting-back-my-domain.md]: typo, gained

File: blog/2019-07-12-getting-back-my-domain.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+Fast forward several years. 
+
+I somehow get into my domain name service (Namecheap) and wire up my domain to point to Github pages. I set up a quick, 
+custom jekyll blog, autopay for my domain every year, and never have to log into the domain name service for any reason. 
+
+Fast forward, later still.
+
+I've gaind maturity as a developer and know my way around cloud technologies (like GCP and AWS). I'm motivated to 
+consolidate my tools on to one platform so I can build more advanced things. 
+To this end, I start to log on to namecheap, which should be easy because I actually remember my username and password.
```
Comment (camerose) [inline blog/2019-07-12-getting-back-my-domain.md]: "...I attempt to log..."

File: blog/2019-07-12-getting-back-my-domain.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+To this end, I start to log on to namecheap, which should be easy because I actually remember my username and password. 
+I enter those pieces of information. Then, I hit the next screen: 
+
+![I have to wait on a text to a phone number I don't control](/assets/namecheap-mfa-masked.png
+ "A MFA screen showing my MaskedMe phone number")
+
+Ok. This is *annoying*, but not too big of a problem. I'll just log into my MaskedMe account, get the multi-factor 
+authentication (MFA) code, and move along.
+
+A google search gets me [here](https://www.abine.com/): Apparently MaskMe has re-branded as Abine, offering two
```
Comment (camerose) [inline blog/2019-07-12-getting-back-my-domain.md]: capitalize Google

File: blog/2019-07-12-getting-back-my-domain.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+
+Eventually, I got a hold of Abine support. They said that after a long enough period of inactivity, they deprecate text
+message proxies. This means that there was no way that I could access the MFA number, and was likely permanently 
+locked out of my Namescheap account. 
+
+This sucked a lot. Not only was I unable to maintain, upgrade, or configure my domain, I risked losing it after my 
+credit card expired and ceased payments. 
+
+The next lever I had left to pull was contacting Namecheap support. In my first conversation with their live help chat 
+service, I gained a glimmer of hope: Their customer service members could override MFA temporarily if I could prove
```
Comment (camerose) [inline blog/2019-07-12-getting-back-my-domain.md]: "gained  a glimmer" to "saw a glimmer"

File: blog/2019-07-12-getting-back-my-domain.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+- I used a series of indirections to remain anonymous... to publicize myself on the internet. 
+- The security measures I took... made me more vulnerable and less in control.
+
+"Congratulations," [my partner](http://camillemerz.com) told me. "You played yourself."
+
+Thanks, hun.
+
+
+Over the next few days, I searched my email and other records for traces of that number with no success. I knew that 
+this was a futile effort, but I did it anyway out of protest or consolation.
```
Comment (camerose) [inline blog/2019-07-12-getting-back-my-domain.md]: desperation?

File: blog/2019-07-12-getting-back-my-domain.md [hunk 8]
```diff
@@ -129,6 +129,11 @@ context to correctly model risk.
 It's easy to ask what I could possibly have been thinking back then. I assure you, I have many times during this whole 
 ordeal. 
 
+I had a misplaced estimation of the orders of magnitude of the tactics I exercised. At the time, I didn't know of the 
+importance of certain moves -- like how [important HTTPS is for web browsing](https://doesmysiteneedhttps.com/). I also 
+neglected to consider the longevity of certain moves. I needed to ask myself, "how dependable are the tools I choose to 
+reply on?"
```
Comment (camerose) [inline blog/2019-07-12-getting-back-my-domain.md]: rely on, not reply on

Comment (alxmrs) [reply blog/2019-07-12-getting-back-my-domain.md]: Thank you!

File: blog/2019-07-12-getting-back-my-domain.html [hunk 9]
```diff
@@ -31,72 +36,58 @@
...
-<p>&quot;That's fine&quot;, I though. &quot;They'll have all my account information in tact.&quot;</p>
-<p>Spoiler: They didn't.</p>
-<p>Apparently, masked credit cards and phone numbers have become part of their <strong>premium</strong> services.</p>
-<p>From here, I took several approaches to recover this proxy cell number. I filed a support ticket (longer wait for free customers). I called their support phone line (a dead end loop that tells you to email them). I even tried to sign up for premium -- for some reason, my attempts to upgrade failed every time (likely because my data was stale, they built their service around new customers instead of maintaining connections for old ones).</p>
+<p>Ok. This is <em>annoying</em>, but not too big of a problem. I’ll just log into my MaskedMe account, get the multi-factor authentication (MFA) code, and move along.</p>
+<p>A Google search gets me <a href="https://www.abine.com/">here</a>: Apparently MaskMe has re-branded as Abine, offering two services: Blur (same services as MaskMe) and DeleteMe, a way to scrub your online footprint off the web.</p>
+<p>“That’s fine”, I though. “They’ll have all my account information in tact.”</p>
```
Comment (camerose) [inline blog/2019-07-12-getting-back-my-domain.html]: thought, not though

File: blog/2019-07-12-getting-back-my-domain.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+
+It's easy to ask what I could possibly have been thinking back then. I assure you, I have many times during this whole 
+ordeal. 
+
+I had a misplaced estimation of the orders of magnitude of the tactics I exercised. At the time, I didn't know of the 
+importance of certain moves -- like how [important HTTPS is for web browsing](https://doesmysiteneedhttps.com/). I also 
+neglected to consider the longevity of certain moves. I needed to ask myself, "how dependable are the tools I choose to 
+rely on?"
+
+I take away from this experience a lesson in skepticism. Before, I thought I needed to be robust against human
```
Comment (awsaavedra) [inline blog/2019-07-12-getting-back-my-domain.md]: What other potential other areas could you improve upon or is it not possible to see yet?

**Pull request merged**